packages/wonder-blocks-testing/src/fixtures/adapters/__tests__/storybook.test.js

```diff
@@ -161,6 +162,45 @@ describe("Storybook Adapter", () => {
                 expect(actionReturnFn).toHaveBeenCalledWith("ARG1", "ARG2");
             });
 
+            it("should inject makeLogHandler function that logs to storybook actions", () => {
+                // Arrange
+                const adapterSpy = jest
+                    .spyOn(AdapterModule, "Adapter")
+                    .mockImplementation(() => {});
+                getAdapter();
+                const actionReturnFn = jest.fn();
+                const actionSpy = jest
+                    .spyOn(AddonActionsModule, "action")
+                    .mockReturnValue(actionReturnFn);
+                const closeGroupFn = adapterSpy.mock.calls[0][1];
+                const getPropsStub = jest.fn();
+                closeGroupFn(
+                    {
+                        title: "TITLE",
+                        description: "DESCRIPTION",
+                    },
+                    null,
+                    [
+                        {
+                            description:
+                                "🎉 This is a story with legal and illegal characters",
+                            component: () => "I AM A COMPONENT",
+                            getProps: getPropsStub,
+                        },
+                    ],
+                );
+                const {logHandler: logHandlerFn} =
+                    getPropsStub.mock.calls[0][0];
+
+                // Act
+                const logFn = logHandlerFn("MESSAGE");
+                logFn("ARG1", "ARG2");
+
+                // Assert
+                expect(actionSpy).toHaveBeenCalledWith("MESSAGE");
+                expect(actionReturnFn).toHaveBeenCalledWith("ARG1", "ARG2");
```

Comment:
**praise:** Nice test.

**commentary:** It's unfortunate that so much set up needs to be done to get things into a scenario where you can test them.  No action req'd.